.travis.yml

```diff
@@ -1,4 +1,12 @@
 language: node_js
+sudo: false
```

Comment:
What's this for? Maybe we want `dist: xenial`?

Reply:
Thought, actually not a fan of specifying a OS version at all.

Reply:
`sudo: false` allows them to use docker, which often leads to faster builds.  We don't need sudo support.  Apparently, that's now the default, so I can remove this.

Reply:
After reading more, I see what you mean about `dist: xenial` (sorry for not reading deeper first!).  I can take any of these approaches you like:

1. leave `sudo: false`
2. remove it, and go back to all defaults
3. add `dist: xenial`

Reply:
If `sudo: false` is the default, I vote for option 2.

Reply:
I think `sudo: false` is only the default with `dist: xenial`, which I think you don't like.  I'll just remove it; worst case we're a tad slower.